index.html

```diff
@@ -12,40 +12,38 @@
     
     <!-- Code will go here for the bootstrap template-->
 
-<!--Navbar-->
-    <nav id="top-navbar" class="navbar navbar-custom">
+    <nav class="navbar navbar-expand-lg navbar-light bg-custom">
```

Comment:
@engle978 I played with this, and I think it be better to have navbar-expand-md  instead of lg.   I feel that the we'd want the burger icon to kick on a smaller resolution than 990px width.   More for devices smaller than a tablet such as phones.

So go ahead and change to **navbar-expand-md**